lib/upfluence/error_logger/sentry.rb

```diff
@@ -20,11 +20,21 @@ def initialize
       end
 
       def notify(error, method, *args)
-        Raven.capture_exception(
-          error,
-          extra: { method: method, arguments: args },
-          tags: { method: method }
-        )
+        begin
+          Raven.capture_exception(
+            error,
+            extra: { method: method, arguments: args },
+            tags: { method: method }
+          )
+        rescue Raven::Error => e
+          if e.message.include? 'the server responded with status 429'
+              Upfluence.logger.info(
+                "Sentry limit exceeded"
+              )
+          else
+            raise e
```

Comment:
What the point of it 🤔 ? just  display a log message (logging level error) would make more sense